packages/support/resources/views/components/input/wrapper.blade.php

```diff
@@ -85,7 +85,7 @@
     @endif
     {{
         $attributes
-            ->except(['wire:target'])
+            ->except(['wire:target', 'tabindex'])
```

Comment:
What is this needed for?

Reply:
This is because the blade compiler duplicates all attributes of the input on the wrapper div including the tabindex which is not the intention and creates another unwanted tab target and steals the focus.